src/App.jsx

```diff
@@ -28,8 +28,24 @@ function App() {
       <p className="read-the-docs">
         Click on the Vite and React logos to learn more
       </p>
+        <ul>
+            <lis></lis>
+            <li>blue blaaaaa<li>
+                <ol>
+                    <li></li>
+                    <li></li>
+                    <li></li>
```

Comment:
Lege <li>-tag zonder inhoud. Overweeg placeholder of verwijder indien niet nodig.